.mailmap

```diff
@@ -96,6 +96,7 @@ Eric Lee <eric.lee@hgst.com>
 Eric Mourgaya <eric.mourgaya@arkea.com> <eric.mourgaya@gmail.com>
 Erwin, Brock A <Brock.Erwin@pnl.gov> <Brock.Erwin@pnl.govgit>
 Esteban Molina-Estolano <eestolan@lanl.gov> <eestolan@29311d96-e01e-0410-9327-a35deaab8ce9>
+Fan Yang <yfceph@gmail.com> <yangf@neunn.com>
```

Comment:
I think you should swap this to make it similar to the entries made by other people.
Fan Yang <yangf@neunn.com> <yfceph@gmail.com>